practice-app/platon_api/rest_api/login/login.py

```diff
@@ -0,0 +1,94 @@
+"""
+Created on MAY 23, 2020
+This script controls the login functionality api of PLATON_API, using django&mysql backend.
+Endpoint description:
+    http://localhost:8000/api/login/
+    
+    'POST':
+        Produces error
+    'GET':
+        Gets dictionary from body.
+        JSON Format : {  
+                        e_mail = "Your email will be here.",
+                        password = "Your password will be here, hashed for protection." 
+                        token = "Your token will be here!!",
+                        }
+@author: Kerem Uslular
+@company: Group7
+"""
+
+from django.shortcuts import render, redirect
+from django.db import connection, transaction
+from platon_api.settings import DATABASES, USER_TABLENAME, WEBSITE_URL
+import re, copy, hashlib, random, json, django.http.request, string
```

Comment:
I think there are some unnecessary imports here. I can not find some of the imports in your code.

Reply:
The lines of code I've used them for since have been deleted. I forgot to update the imports. I will change them accordingly.